articles/flows/guides/device-auth/includes/refresh-tokens.md

```diff
@@ -0,0 +1,51 @@
+## Refresh Tokens
+
+You have already received a [Refresh Token](/tokens/refresh-token/current) if you've been following this tutorial and completed the following:
+
+* configured your API to allow offline access
+* included the `offline_access` scope when you initiated the authentication request through the [authorize](/api/authentication/reference#authorize-application) endpoint
```

Comment:
We need to add the endpoint to the explorer https://auth0team.atlassian.net/browse/IPS-378 (maybe iam-protocols-sessions can do that with your help)

Reply:
Done.